tests/components/evil_genius_labs/conftest.py

```diff
@@ -0,0 +1,46 @@
+"""Test helpers for Evil Genius Labs."""
+import json
+import pathlib
+from unittest.mock import patch
+
+import pytest
+
+from homeassistant.setup import async_setup_component
+
+from tests.common import MockConfigEntry
+
+
+@pytest.fixture(scope="session")
+def data_fixture():
+    """Fixture data."""
+    data = json.loads(
+        (pathlib.Path(__file__).parent / "evil-genius-labs-data.json").read_text()
```

Comment:
We have a test helper to load fixtures.

https://github.com/home-assistant/core/blob/a90c8ab5584ce50e8dfae67790cb56f3d1d1ea86/tests/common.py#L400

Reply:
Having all fixtures for all tests live in a single folder instead of being in same directory as the test is leftover behavior from when platforms lived in integration folders (light/hue.py). I think that we should move to this approach instead, and in that case the helper doesn't help.

Reply:
Ok. Then we should move the fixtures to a subdirectory `fixtures` in the integration test directory.

Reply:
We could extend the helper with an optional parent dir parameter.

Reply:
This adds a fixture loader helper for integrations: https://github.com/home-assistant/core/pull/58902/files

Will rebase this PR once that is merged.